src/main.rs

```diff
@@ -12,13 +13,17 @@ use std::io::Read;
 use colored::Colorize;
 use structopt::StructOpt;
 
-#[derive(StructOpt, Debug)]
+#[derive(StructOpt, Debug, Clone)]
 #[structopt(name = "bingrep", about = "bingrep - grepping through binaries since 2017")]
 struct Opt {
     /// A flag, true if used in the command line.
     #[structopt(short = "d", long = "debug", help = "Print debug version of parse results")]
     debug: bool,
 
+    /// A flag, true if used in the command line.
+    #[structopt(short = "D", long = "demangle-rust", help = "Apply Rust demangling")]
```

Comment:
I think this will demangle c++ symbols as well, so I'd prefer long to be --demangle. And if not, still prefer since the way you set it up we can enable that later on.

Thanks and besides this last nit, great work, LGTM, thanks so much!!! :heart_eyes: